File: app/src/main/java/com/tournamentbracketjava/Contracts/BracketInterface.java
package com.tournamentbracketjava.Contracts;

import com.tournamentbracketjava.Models.Annotations;
import com.tournamentbracketjava.Models.Connections;
import com.tournamentbracketjava.Models.Rows;
import java.util.List;

public interface BracketInterface {

     interface View {
         void setRows();
         void setAnnotations();
         void setConnections();
         void setWinningTeam();
    }

     interface Model {
         List<Rows> getRows();
         List<Annotations> getAnnotations();
         List<Connections> getConnections();
         int getWinningTeam();
    }
}

File: app/src/main/res/layout/com/bracket/Contracts/BracketInterface.kt
package com.bracket.Contracts

import com.bracket.Models.Annotations
import com.bracket.Models.Connections
import com.bracket.Models.Rows

interface BracketInterface {

    interface View {
        fun setRows()
        fun setAnnotations()
        fun setConnections()
        fun setWinningTeam()
    }

    interface Model {
        fun getRows(): List<Rows>
        fun getAnnotations(): List<Annotations>
        fun getConnections(): List<Connections>
        fun getWinningTeam(): Int
    }

}